google/cloud/sql/connector/refresh_utils.py

```diff
@@ -229,3 +231,34 @@ async def _is_valid(task: asyncio.Task) -> bool:
         # supress any errors from task
         logger.debug("Current instance metadata is invalid.")
     return False
+
+
+def _downscope_credentials(
+    credentials: Credentials,
+    scopes: List[str] = ["https://www.googleapis.com/auth/sqlservice.login"],
```

Comment:
Do we scope to just `sqladmin` for the other API calls? (if not, probably should start doing it)

Reply:
The default credentials are created with the following scopes within [`_auth_init`](https://github.com/GoogleCloudPlatform/cloud-sql-python-connector/blob/main/google/cloud/sql/connector/instance.py#L264-L267):
```python
scopes = [
    "https://www.googleapis.com/auth/sqlservice.admin",
    "https://www.googleapis.com/auth/cloud-platform",
]
```

Are you saying we may only need `"https://www.googleapis.com/auth/sqlservice.admin"` and not the second scope?

Reply:
Yes, we shouldn't need cloud-platform

Reply:
removed the scope